packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -471,12 +471,12 @@ class IOSSimulator extends Device {
       printTrace('Waiting for observatory port to be available...');
 
       try {
-        int devicePort = await observatoryDiscovery
-          .nextPort()
+        Uri deviceUri = await observatoryDiscovery
+          .nextUri()
           .timeout(new Duration(seconds: 20));
-        printTrace('service protocol port = $devicePort');
-        printStatus('Observatory listening on http://127.0.0.1:$devicePort');
-        return new LaunchResult.succeeded(observatoryPort: devicePort);
+        printTrace('service protocol = $deviceUri');
+        printStatus('Observatory listening on $deviceUri');
```

Comment:
I know that some IDEs are (or were) scraping stdout for `Observatory listening on ...`. I'm going to leave it as a `printStatus` until I better understand if that change will break clients.